exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/OperatorWrapper.java

```diff
@@ -56,8 +55,29 @@
   private final CoreOperatorType operatorType;
   private final String operatorName;
   private final int size;
+  private static AtomicBoolean initializedWarningThreshold = new AtomicBoolean();
+  private static int timeSkewMin;
+  private static double timeSkewRatio;
+  private static int scanWaitMin;
+  private static double waitSkewRatio;
+
+  //One time server setting for setting threshold warnings
+  static void setWarningThresholds(DrillConfig config) {
```

Comment:
Please to do use static, do assignment as for regular class variables. Currently DrillConfig does not change but this might change (and even if not), statics is not the best way / practice here plus you accept DrillConfig in the constructor which already implicates you use it for class variables.